inst/markdown/compareScenarios2/cs2_06_energy_services.Rmd

```diff
@@ -206,11 +206,37 @@ stock <- data %>%
 showLinePlots(stock, "Production|Industry|Steel|Accumulated Production", mainReg = "World")
 
 stock_pc <- left_join(stock, population, by = c("period", "region", "scenario")) %>%
-  mutate(value = value.x / value.y, unit = "t per cap")
+  mutate(value = value.x / value.y, unit = "t per cap") %>% 
+  select(-value.x,-value.y)
 
 showLinePlots(stock_pc, "Production|Industry|Steel|Accumulated Production", scales = "fixed", mainReg = "World")
```

Comment:
I would go with "cumulated production".  "Accumulated" is too suggesting too strongly that this is the accumulation of steel in the economy, for which the depreciation is missing.